intellij/src/saros/intellij/ui/Messages.java

```diff
@@ -85,10 +85,12 @@
   public static String ConnectButton_connect_to_new_account_title;
   public static String ConnectButton_connect_to_new_account_message;
 
-  public static String NegotiationHandler_canceled_invitation;
-  public static String NegotiationHandler_canceled_invitation_text;
-  public static String NegotiationHandler_error_during_invitation;
-  public static String NegotiationHandler_error_during_invitation_text;
+  public static String NegotiationHandler_session_local_error_title;
```

Comment:
I do not know if it is a good decision to mix session_negotiation and invention. Just stick to one wording (I do not prefer session negotiation and not invitation)

Reply:
Done.